bug: Insufficient debug logging from ginkgo E2E test in CI

### Describe the bug

I got a failure of the ginkgo E2E test in CI. See https://github.com/kubestellar/kubestellar/actions/runs/9476682004/job/26109961479?pr=2199 . The run of setup-kubestellar.sh suffered a timeout. The CI test log had no indication of **where** in the running of setup-kubestellar.sh the timeout happened.

Here is what I got in the log:

```
Run cd test/e2e/ginkgo 
  cd test/e2e/ginkgo 
  KFLEX_DISABLE_CHATTY=true ginkgo -vv --trace --no-color $TEST_FLAGS
  shell: /usr/bin/bash -e {0}
  env:
    KO_DOCKER_REPO: ghcr.io/kubestellar/kubestellar
    TEST_FLAGS: 
go: downloading github.com/onsi/gomega v1.29.0

Running Suite: Kubestellar testing - /home/runner/work/kubestellar/kubestellar/test/e2e/ginkgo
==============================================================================================
Random Seed: 1718166228

Will run 17 of 17 specs
------------------------------
[BeforeSuite] 
/home/runner/work/kubestellar/kubestellar/test/e2e/ginkgo/ginkgo_suite_test.go:68
  > Enter [BeforeSuite] TOP-LEVEL - /home/runner/work/kubestellar/kubestellar/test/e2e/ginkgo/ginkgo_suite_test.go:68 @ 06/12/24 04:24:27.163
  STEP: Execing command [../common/setup-kubestellar.sh] - /home/runner/work/kubestellar/kubestellar/test/util/util.go:126 @ 06/12/24 04:24:28.008
  [TIMEDOUT] A suite timeout occurred
  In [BeforeSuite] at: /home/runner/work/kubestellar/kubestellar/test/e2e/ginkgo/ginkgo_suite_test.go:68 @ 06/12/24 05:23:48.889

  Full Stack Trace

  This is the Progress Report generated when the suite timeout occurred:
    In [BeforeSuite] (Node Runtime: 59m21.726s)
      /home/runner/work/kubestellar/kubestellar/test/e2e/ginkgo/ginkgo_suite_test.go:68
      At [By Step] Execing command [../common/setup-kubestellar.sh] (Step Runtime: 59m20.881s)
        /home/runner/work/kubestellar/kubestellar/test/util/util.go:126

      Spec Goroutine
      goroutine 19 [syscall, 59 minutes]
        syscall.Syscall6(0xc000186000?, 0xc00049a060?, 0xc0000dfd06?, 0xc0000dfe28?, 0x55e34e?, 0x1854965?, 0x1?)
          /opt/hostedtoolcache/go/1.20.14/x64/src/syscall/syscall_linux.go:91
        os.(*Process).blockUntilWaitable(0xc0004d4060)
          /opt/hostedtoolcache/go/1.20.14/x64/src/os/wait_waitid.go:32
        os.(*Process).wait(0xc0004d4060)
          /opt/hostedtoolcache/go/1.20.14/x64/src/os/exec_unix.go:22
        os.(*Process).Wait(...)
          /opt/hostedtoolcache/go/1.20.14/x64/src/os/exec.go:132
        os/exec.(*Cmd).Wait(0xc000186000)
          /opt/hostedtoolcache/go/1.20.14/x64/src/os/exec/exec.go:890
        os/exec.(*Cmd).Run(0x1854965?)
          /opt/hostedtoolcache/go/1.20.14/x64/src/os/exec/exec.go:590
      > github.com/kubestellar/kubestellar/test/util.SetupKubestellar(0x0)
          /home/runner/work/kubestellar/kubestellar/test/util/util.go:130
            | cmd.Stderr = &e
            | cmd.Stdout = &o
            > err := cmd.Run()
            | if err != nil {
            | 	fmt.Fprintf(ginkgo.GinkgoWriter, "%s", o.String())
      > github.com/kubestellar/kubestellar/test/e2e/ginkgo.glob..func1()
          /home/runner/work/kubestellar/kubestellar/test/e2e/ginkgo/ginkgo_suite_test.go:71
            | if !skipSetupFlag {
            | 	util.Cleanup()
            > 	util.SetupKubestellar(releasedFlag)
            | }
            | 
        github.com/onsi/ginkgo/v2/internal.extractBodyFunction.func3({0x0, 0x0})
          /home/runner/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.13.0/internal/node.go:463
        github.com/onsi/ginkgo/v2/internal.(*Suite).runNode.func3()
          /home/runner/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.13.0/internal/suite.go:889
        github.com/onsi/ginkgo/v2/internal.(*Suite).runNode
          /home/runner/go/pkg/mod/github.com/onsi/ginkgo/v2@v2.13.0/internal/suite.go:876
  < Exit [BeforeSuite] TOP-LEVEL - /home/runner/work/kubestellar/kubestellar/test/e2e/ginkgo/ginkgo_suite_test.go:68 @ 06/12/24 05:23:48.89 (59m21.727s)
[BeforeSuite] [TIMEDOUT] [3561.727 seconds]
[BeforeSuite] 
/home/runner/work/kubestellar/kubestellar/test/e2e/ginkgo/ginkgo_suite_test.go:68
------------------------------

Summarizing 1 Failure:
  [TIMEDOUT] [BeforeSuite] 
  /home/runner/work/kubestellar/kubestellar/test/e2e/ginkgo/ginkgo_suite_test.go:68

Ran 0 of 17 Specs in 3561.727 seconds
FAIL! - Suite Timeout Elapsed -- A BeforeSuite node failed so all tests were skipped.
--- FAIL: TestGinkgo (3561.73s)
FAIL

Ginkgo ran 1 suite in 1h0m0.022071824s

Test Suite Failed
Error: Process completed with exit code 1.
```


### Steps To Reproduce

See abovve

### Expected Behavior

setup-kubestellar.sh prints a lot of stuff. Seeing that would be very helpful.

### Additional Context

_No response_

--- a/test/util/util.go
+++ b/test/util/util.go
@@ -17,7 +17,6 @@ limitations under the License.
 package util
 
 import (
-	"bytes"
 	"context"
 	goerrors "errors"
 	"fmt"
@@ -112,21 +111,15 @@ func CreateNS(ctx context.Context, client *kubernetes.Clientset, name string) {
 
 func Cleanup(ctx context.Context) {
 	ginkgo.GinkgoHelper()
-	var e, o bytes.Buffer
 	cmd := exec.CommandContext(ctx, "../common/cleanup.sh")
-	cmd.Stderr = &e
-	cmd.Stdout = &o
+	cmd.Stderr = ginkgo.GinkgoWriter
+	cmd.Stdout = ginkgo.GinkgoWriter
 	err := cmd.Run()
-	if err != nil {
-		fmt.Fprintf(ginkgo.GinkgoWriter, "%s", o.String())
-		fmt.Fprintf(ginkgo.GinkgoWriter, "%s", e.String())
-	}
 	gomega.Expect(err).To(gomega.Succeed())
 }
 
 func SetupKubestellar(ctx context.Context, releasedFlag bool, otherFlags ...string) {
 	ginkgo.GinkgoHelper()
-	var e, o bytes.Buffer
 	var args []string
 	if releasedFlag {
 		args = []string{"--released"}
@@ -135,13 +128,9 @@ func SetupKubestellar(ctx context.Context, releasedFlag bool, otherFlags ...stri
 	commandName := "../common/setup-kubestellar.sh"
 	ginkgo.By(fmt.Sprintf("Execing command %#v", append([]string{commandName}, args...)))
 	cmd := exec.CommandContext(ctx, commandName, args...)
-	cmd.Stderr = &e
-	cmd.Stdout = &o
+	cmd.Stderr = ginkgo.GinkgoWriter
+	cmd.Stdout = ginkgo.GinkgoWriter
 	err := cmd.Run()
-	if err != nil {
-		fmt.Fprintf(ginkgo.GinkgoWriter, "%s", o.String())
-		fmt.Fprintf(ginkgo.GinkgoWriter, "%s", e.String())
-	}
 	gomega.Expect(err).To(gomega.Succeed())
 }
 

